garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -403,7 +403,7 @@ function CORPSE.Create(ply, attacker, dmginfo)
    rag.player_ragdoll = true
    rag.sid = ply:SteamID()
 
-   rag.uqid = ply:UniqueID() -- backwards compatibility; use rag.sid instead
+   rag.uqid = rag.sid

```

Comment:
Why even change this? The sid key was already added for this reason.

Reply:
Wasn't sure on that, really, but there are no references to it (in here, at least). That comment tho. Perhaps I should have used AccountID there.

Reply:
It's used by addons that identify ragdolls by UniqueID. All of those instances would have to change.